inject.js

```diff
@@ -1,6 +1,6 @@
   var tc = {
     settings: {
-      speed: 1.0,           // default 1x
+      lastSpeed: 1.0,           // default 1x
```

Comment:
Hmm, why do we need to change the name?

Reply:
With the new dict speeds, the remaining purpose of the variable previously called speed is to track the most recent speed for the purposes of people who have "remember your speed" checked. It is confusing to have a variable named speed and speeds so I thought we should rename the variable speed to better capture its current purpose.

Reply:
Gotcha, makes sense.